sourced/ml/core/models/id_splitter.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import string
+from typing import Dict, List, Sequence, Tuple
+
+from modelforge import Model, register_model
+import numpy
+from tensorflow import keras
+from tensorflow.keras.preprocessing.sequence import pad_sequences
+
+from sourced.ml.core.algorithms.id_splitter.nn_model import (f1score, precision,
+                                                             recall)
+from sourced.ml.core.models.license import DEFAULT_LICENSE
+
+
+@register_model
+class IdentifierSplitterBiLSTM(Model):
+    """
+    Bidirectionnal LSTM Model. Splits identifiers without need for a conventional pattern.
```

Comment:
```suggestion
    Bidirectional LSTM Model. Splits identifiers without need for a conventional pattern.
```
I wonder if I missed this last time or it was added afterward...

Reply:
Probably my bad. Fixed.